docs/examples/searchguidelink/avatarExample.tsx

```diff
@@ -0,0 +1,76 @@
+import { Avatar, AvatarGroup, Flex, Icon, Image, SearchGuideLink } from 'gestalt';
```

Comment:
Can we shod all this permutations?

Actually instead of all in one, can we create one example for Avtar, icon and image and have on each the 3 variants?

![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/ddfa7006-4bd8-4486-800e-ac1fb6c820c8)
![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/0bf4bf93-e310-4b34-b143-a8b1b1cbd65c)
![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/db016d42-b0e3-4d7b-87ce-7b785a1f7464)
![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/25c88285-1ef3-43fe-a837-0b16abdbe4b8)


![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/7a319258-12c7-47e2-840c-17ca8ef2fa03)